custom/icds_reports/tasks.py

```diff
@@ -791,6 +793,17 @@ def collect_inactive_awws():
     celery_task_logger.info("Ended updating the Inactive AWW")
 
 
+@periodic_task(run_every=crontab(hour=23, minute=0, day_of_month='20'), acks_late=True, queue='icds_aggregation_queue')
```

Comment:
@emord  @calellowitz 

In a given month (after 5th), only last month's data will be provided via the API, so I have put this task on 20th. Not sure if this is the right time/place to trigger this. Let me know if this should go somewhere else